packages/compass-global-writes/src/components/states/sharding.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {
+  Banner,
+  BannerVariant,
+  Body,
+  css,
+  Link,
+  spacing,
+} from '@mongodb-js/compass-components';
+import { connect } from 'react-redux';
+
+const nbsp = '\u00a0';
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[400],
+});
+
+export function ShardingState() {
+  return (
+    <div className={containerStyles}>
+      <Banner variant={BannerVariant.Info}>
+        <strong>Sharding your collection ...</strong>
```

Comment:
```suggestion
        <strong>Sharding your collection …</strong>
```

(or I guess you can &hellip; like the &apos; below – but just fyi, &apos; is just a literal `'` that you don't really need to escape here anyway. And even more pedantically, you'd want to use `‘ ’` instead of `'` 😄)